crates/oxc_linter/src/snapshots/no_useless_spread.snap

```diff
@@ -0,0 +1,677 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: no_useless_spread
+---
+  ⚠ eslint-plugin-unicorn(no-useless-spread): Using a spread operator here creates a new array unnecessarily.
+   ╭─[no_useless_spread.tsx:1:1]
+ 1 │ const array = [...[a]]
+   ·                   ───
```

Comment:
We don't have a span to `...`, but you can probably get it from the first 3 chars of the `SpreadElement`.

Reply:
good call